core/templates/dev/head/pages/exploration_editor/ExplorationSaveService.js

```diff
@@ -93,6 +94,17 @@ oppia.factory('ExplorationSaveService', [
             };
             $scope.explorationId = (
               ExplorationContextService.getExplorationId());
+            $scope.explorationLink = $window.location.protocol + '//' +
+            $window.location.host + '/explore/' + $scope.explorationId;
```

Comment:
For things that span multiple lines, format as follows (note parens and indentation):

```
           $scope.explorationLink = (
             $window.location.protocol + '//' +
             $window.location.host + '/explore/' + $scope.explorationId);
```

Reply:
Done, thanks!